indexing-service/src/main/java/io/druid/indexing/common/TaskToolboxFactory.java

```diff
@@ -75,7 +76,6 @@ public TaskToolboxFactory(
       DataSegmentMover dataSegmentMover,
       DataSegmentArchiver dataSegmentArchiver,
       DataSegmentAnnouncer segmentAnnouncer,
-      FilteredServerView newSegmentServerView,
```

Comment:
minor nit, but we should keep the arguments in the same order and just replace newSegmentServerView with handoffNotifierFactory. Makes it easier to track changes and resolve conflicts.